pyrene/src/components/TabView/TabView.jsx

```diff
@@ -189,11 +191,13 @@ TabView.propTypes = {
    */
   tabHeaderElement: PropTypes.element,
   /**
-   * Data input array for the tabs. Type: [{ name: string (required), renderCallback: func (required), disabled: bool }]
+   * Data input array for the tabs.
+   * Type: [{ name: string (required), nameRenderCallback: func, renderCallback: func (required), disabled: bool }]
```

Comment:
`renderCallback: func (required)`, `required` is not correct, is it?

Reply:
ood question! let me check :)
Ah, looks like it is indeed required:
`renderCallback: PropTypes.func.isRequired,` :)

Reply:
Sorry, my mistake. Confused renderCallback and nameRenderCallback.